src/rsz/src/RepairSetup.cc

```diff
@@ -641,6 +638,27 @@ RepairSetup::upsizeCell(LibertyPort *in_port,
   return nullptr;
 }
 
+Point RepairSetup::computeCloneGateLocation(const Pin *drvr_pin,
+                              const vector<pair<Vertex*, Slack>> &fanout_slacks)
+{
+  int count(1); // driver_pin counts as one
+  int centroid_x(0), centroid_y(0); // (0, 0)
+
+  centroid_x += db_network_->location(drvr_pin).getX();
+  centroid_y += db_network_->location(drvr_pin).getY();
+
+  int split_index = fanout_slacks.size() / 2;
```

Comment:
Is this fanout splitting arbitrary?

Reply:
I don't see a response but this is marked resolved.

Reply:
github somehow lost my comment. The fanout list is sorted.

Reply:
It is sorted by slack not spatial proximity afaik.  It seems odd to take the centroid of such.  I'll approve but I suspect there is further work to do here.